### ISSUE TITLE
Copy of Move Celery Main to Helmfile/AWS Secrets Manager

### ISSUE THREAD

**Issue by P0NDER0SA** [cds-snc/notification-planning-core#466]
# Description

As a developer of notify, I would like all of our kubernetes deployments to be managed by Helmfile so that all of our kubernetes code is in one location.  I would also like to have all secrets automated from a single source of truth (1pass -> AWS Secrets) and remove our dependency on encrypted .env files

## WHY are we building?

- Provides the ability to deploy dynamic environments
- Provides the ability to run "diff" commands against environments
- Reduces the amount of code that Notify has to manage
- Keeps code DRY

## WHAT are we building?

- [ ] Add Document Download entry to the helmfile
- [ ] source secrets for helmfile from AWS Secrets Manager
- [ ] source configuration values from helmfile overrides where necessary
- [ ] adjust GitHub actions to work with these changes
- [ ] This work is scoped to the DEV environment.

## VALUE created by our solution

- Increased reliability via reduced chance of accidental changes during releases
- Increased velocity due to reduction in maintenance
- better secrets management that falls in line with our OKRs

# Acceptance Criteria

- [ ] Celery Main are deployed and managed with helmfile
- [ ] All secrets required by Document Download are sourced from 1pass -> terraform -> aws secrets manager and accessed using a kubernetes secret
- [ ] Celery Main still works

# Not in scope

- [ ] Deployment in staging and production: we should only deploy to dev until we figure out the deployment steps.

# QA Steps

- [ ] verify that the Celery Pods are functional and running properly (might need some guidance on the best way to QA this!)

**Comment by P0NDER0SA** [comment]
Began work on this.  I have a couple of different ways I could handle it, so I will have a conversation with the team to decide which route i'll take.  I've already bootstrapped the project so it's ready for migration.

**Comment by P0NDER0SA** [comment]
https://github.com/cds-snc/notification-manifests/pull/3137

Draft PR

**Comment by jimleroyer** [comment]
Currently blocked by release of other components, so waiting on other tasks.

**Comment by P0NDER0SA** [comment]
Starting work on this -- branch helm-notify-system-migration

**Comment by sastels** [comment]
pushed a broken migration to staging manually, new log line does not appear :/ Will investigate further...

**Comment by ben851** [comment]
Ben & Pond to chat today

**Comment by sastels** [comment]
Have a [WIP PR](https://github.com/cds-snc/notification-manifests/pull/3403) - more work than we thought. Will roll out piece by piece.
